docs/docs/adding-page-transitions-with-plugin-transition-link.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Using transition-link for Page Transitions
```

Comment:
Same as above: let's use the full plugin name `Using gatsby-plugin-transition-link for Page Transitions`